src/main/resources/persistence.properties

```diff
@@ -0,0 +1,17 @@
+hibernate.dialect = org.hibernate.dialect.PostgreSQL9Dialect
+hibernate.connection.driver_class = org.postgresql.Driver
+hibernate.archive.autodetection = class
+
+javax.persistence.jdbc.url = jdbc:postgresql://188.166.92.29:5432/moodcat_db
+javax.persistence.jdbc.user = moodcat
+javax.persistence.jdbc.password = INSERT_POSTGRES_PASSWORD
```

Comment:
Be sure to set the postgres password on your local system! I've added `src/main/resources/persistence.properties` to the `.gitignore` to prevent the password from bubbling into the repository.